File: DayTwo/Python/book_suggestion_system_main_app.py
from book_suggestion_system import *

while True:
    print(""" 
    Welcome to the Book Suggestion System!
    1. Get Suggestions
    2. Add Book
    3. Remove Book
    4. Update book
    5. Show all books
    6. Exit

""")

    user_input = int(input("Choose an operation: "))
 
    match user_input:
        
        case 1:
            print("Book for the Day:\nBook Title: ", end ="")
            print(get_book_Suggestion())
            print("Page: ", end ="")
            print(get_page_number())
            
            another_suggestion = input("Would you like to get another suggestion? (yes/no): ")
            
            while (another_suggestion.lower() == "yes"):
            
                print("Book for the Day:\nBook Title: ", end ="")
                print(get_book_Suggestion())
                print("Page: ", end ="")
                print(get_page_number())
                
                another_suggestion = input("Would you like to get another suggestion? (yes/no): ")
                
        case 2:
            
            book_title = input("Enter the book title: ")
            add_new_books(book_title)
            print("Book added successfully!")
            
        case 3:

            book_title = input("Enter the book title to remove: ")
            remove_books(book_title)
            print("Book removed successfully!")
            
            
        case 4:

            old_book_title = input("Enter the old book title: ")
            new_book_title = input("Enter the new book title: ")
            update_books(old_book_title, new_book_title)
            print("Book updated successfully!")
            
        case 5:
            print("All Books")
            count = 1
            for elements in show_books():
                print(f"{count}. {elements}")
                count+=1
            
        case 6:
            break
                
                
        

File: DayTwo/JavaScript/bookSuggestionSystemMainApp.js
const prompt = require("prompt-sync")();
const {getBookSuggestion, getPageNumber, addNewBooks, removeBooks, updateBooks, showBooks} = require("./bookSuggestionSystem.js");



while(true){

    console.log(` 
    Welcome to the Book Suggestion System!
    1. Get Suggestions
    2. Add Book
    3. Remove Book
    4. Update book
    5. Show all books
    0. Exit

    `)


    const userInput = Number(prompt("Choose an operation: "))
     
    if(userInput == 0)
        break;
            
    switch(userInput){
            
        case 1:
            process.stdout.write(`Book for the Day:\nBook Title: `)
            console.log(getBookSuggestion())
            process.stdout.write("Page: ")
            console.log(getPageNumber())
            
            let anotherSuggestion = prompt("Would you like to get another suggestion? (yes/no): ")
            
            while (anotherSuggestion.toLowerCase() == "yes"){
            
                process.stdout.write("Book for the Day:\nBook Title: ")
                console.log(getBookSuggestion())
                process.stdout.write("Page: ")
                console.log(getPageNumber())
                
                anotherSuggestion = prompt("Would you like to get another suggestion? (yes/no): ")

            }break;
            
        case 2:
            
            const bookTitle = prompt("Enter the book title: ");
            addNewBooks(bookTitle);
            console.log("Book added successfully!");
            break;
            
        case 3:

            const bookName = prompt("Enter the book title to remove: ");
            removeBooks(bookName);
            console.log("Book removed successfully!");
            break;
            
        case 4:

            const oldBookTitle = prompt("Enter the old book title: ");
            const newBookTitle = prompt("Enter the new book title: ");
            updateBooks(oldBookTitle, newBookTitle);
            console.log("Book updated successfully!");
            break;
            
        case 5:
            console.log("All Books");
            let count = 1;
            for (let book of showBooks()){
                console.log(`${count}. ${book}`)
                count+=1;
            }
            break;


    }


}














         

                
                
        
